docs/api/app.md

```diff
@@ -533,17 +533,16 @@ You can request the following paths by the name:
 * `logs` Directory for your app's log folder.
 * `pepperFlashSystemPlugin` Full path to the system version of the Pepper Flash plugin.
 
-### `app.getFileIcon(path[, options], callback)`
+### `app.getFileIcon(path[, options])`
 
 * `path` String
 * `options` Object (optional)
   * `size` String
     * `small` - 16x16
     * `normal` - 32x32
     * `large` - 48x48 on _Linux_, 32x32 on _Windows_, unsupported on _macOS_.
-* `callback` Function
-  * `error` Error
-  * `icon` [NativeImage](native-image.md)
+
+Returns `Promise<icon>` - fulfilled with the app's icon, which is a [NativeImage](native-image.md).
```

Comment:
`Promise<icon>` looks like a type notation, so it would be more natural to make it `Promise<NativeImage>`.